appeals/web/src/server/appeals/appeal-details/interested-party-comments/__tests__/interested-party-comments.test.js

```diff
@@ -182,4 +181,43 @@ describe('interested-party-comments', () => {
 			expect(invalidMessage?.textContent?.trim()).toBe('No invalid comments');
 		});
 	});
+
+	describe('GET /review-comment with data', () => {
+		beforeEach(() => {
+			nock('http://test/').get('/appeals/2/reps/5').reply(200, interestedPartyCommentForReview);
+		});
+
+		it('should render review comment page with the provided comment details', async () => {
+			const response = await request.get(`${baseUrl}/2/interested-party-comments/5/review`);
+
+			expect(response.statusCode).toBe(200);
+
+			const dom = parseHtml(response.text);
+			const elementInnerHtml = dom.innerHTML;
+			expect(elementInnerHtml).toMatchSnapshot();
+			expect(elementInnerHtml).toContain('Review comment</h1>');
+
+			const interestedPartyRow = dom.querySelectorAll('.govuk-summary-list__row')[0];
```

Comment:
This is looking great - just FYI you can use `parseHtml` to select specific elements from the markup, without having to use `querySelectorAll`. For example: ```const element = parseHtml(response.text, {
				rootElement: notificationBannerElement
			});```